website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,169 @@
+
+Podman Desktop 1.7 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Podman 4.9
+
+🦭 Podman 4.9 includes key fixes for stability and reliability problems reported by our users.
+If you've been floundering we highly recommend updating!
+
+### Extension API Improvements
+
+We spent a lot of time this release adding new extension API to give upcoming extensions
+more capability and even better integration into 🦭 Podman Desktop. We've added support
+for full page webviews, image badges, icons, a navigation API, and API access to more function
+from the container engine:
+
+- Webview in the UI [#5594](https://github.com/containers/podman-desktop/pull/5594)
+- Add webview API for extensions [#5592](https://github.com/containers/podman-desktop/pull/5592)
+- Allow extensions to list webviews [#5628](https://github.com/containers/podman-desktop/pull/5628)
+- Create container without starting it [#5643](https://github.com/containers/podman-desktop/pull/5643)
+- Expose create/start Pod and replicatePodmanContainer [#5648](https://github.com/containers/podman-desktop/pull/5648)
+- Expose create/list/delete volumes for extensions [#5598](https://github.com/containers/podman-desktop/pull/5598)
+- Add getImageInspect to API [#5596](https://github.com/containers/podman-desktop/pull/5596)
+- New contribution points for icon of image [#5543](https://github.com/containers/podman-desktop/pull/5543)
+- Add BuildOption [#5533](https://github.com/containers/podman-desktop/pull/5533)
+- Add platform parameter to image build method [#5501](https://github.com/containers/podman-desktop/pull/5501)
+- Expose build image method [#5500](https://github.com/containers/podman-desktop/pull/5500)
+- Navigation api [#5558](https://github.com/containers/podman-desktop/pull/5558)
+- Register badges by extensions for image list/details [#5557](https://github.com/containers/podman-desktop/pull/5557)
+- Install extensions from private registries [#5473](https://github.com/containers/podman-desktop/pull/5473)
+
+### Experimental Kubernetes UI
+
+We have been working the last couple months to expand our support for Kubernetes. This support isn't ready
+to set sail yet, but if you're working with a Kubernetes cluster we'd love to start getting your feedback
+on the direction!
+
+To enable the new support, go to Settings > Preferences > Kubernetes, and enable the Experimental option:
+
+![Kubernetes Preference](img/podman-desktop-release-1.7/kube-preference.png)
+
+This will add three new items to the main navigation, allowing you to view
+Deployments, Services, and Ingress & Routes:
+
+![Kubernetes Deployments](img/podman-desktop-release-1.7/kube-deployments.png)
+![Kubernetes Services](img/podman-desktop-release-1.7/kube-services.png)
+![Kubernetes Ingresses & Routes](img/podman-desktop-release-1.7/kube-ingress-routes.png)
+
+In the 1.7 release you can view details of deployments and services, but not ingresses or routes.
```

Comment:
Seems opposite to what is said at line 72

Reply:
We have summary pages for all 4, but you can only click to go to Details pages for the first two. I re-wrote this a bit and tried to explain, let me know if it's still not clear.